### ISSUE TITLE
signal: killed error when using go get to get mesos-dns

### ISSUE THREAD

**Issue by F21** [mesosphere/mesos-dns#163]
I am trying to build mesos-dns using these instructions: http://docs.mesosphere.com/intro-course/ex7.html

My go is version `go version go1.4.2 linux/amd64` and OS is `CentOS 7.1 Linux node1 3.10.0-229.el7.x86_64 #1 SMP Fri Mar 6 11:36:42 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux`.

This is what happens when I try to use `go get` to get `mesos-dns`:
```
[vagrant@node1 ~]$ go get github.com/mesosphere/mesos-dns
go build github.com/mesos/mesos-go/mesosproto: /usr/lib/golang/pkg/tool/linux_amd64/6g: signal: killed
```

For some reason, `go get` will take a very long time, and then terminate with a signal killed error.

**Comment by gomes** [comment]
Hi @F21 I did the tutorial using a machine like yours (CentOS 7.1.503 and Kernel 3.10.0-229.el7.x86_64). All works well. Can you check your connection and connectivity with github.com?

**Comment by F21** [comment]
I believe this is due to the lack of memory. I increased the memory of my vagrant boxes to 1GB and it seems to be fine.

**Comment by dj80hd** [comment]
@F21 - Thank you.  Increasing memory to 1G worked.

**Comment by seglo** [comment]
I ran into a similar problem, but only while debugging why my ansible playbook as part of exercise 16.  The playbook executed successfully, yet it's clear it didn't install correctly.  When I did some more troubleshooting the mesos-dns executable didn't exist and when attempting to build it manually ran into this problem.  I've since changed my `Vagrantfile` and rebuilt my cluster with 1024MB RAM and everything worked.

Thanks @gomes
